compiler/src/test/scala/dx/compiler/CompilerTest.scala

```diff
@@ -973,6 +978,14 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
         )
     )
 
+    if (desc1.access == expected2Access && desc2.access == expected1Access) {
+      fail("Compiled applet order is reversed")
+    }
+
+    desc1.access shouldBe expected1Access
+
+    desc2.access shouldBe expected2Access
+
```

Comment:
I dont get it why the access order would be wrong. The parallelized compilation is done on callable, and the network access is embedded in the callable object, so it would go with the callable and should not be affected by this new setting. @r-i-v-a

Reply:
@YuxinShi0423 I think the test is just asserting that the new setting doesn't change the order; this was mentioned in the spec attached to the PMUX ticket.

Reply:
I had introduced a bug that _did_ reverse the order, but the resulting error message was unclear (it looked like the results were just wrong). I made the message for reversed order more clear and fixed the bug.

Reply:
@TedBrookings could you point me to the commit for that fix?

Reply:
Thanks! So the access config is just the flag we used to check if applet1 and 2 are in correct order, but itself wont not be swapped.  
I would still prefer another unit test since this one serves two purposes now. But I am okay with it. @r-i-v-a